### DISCUSSION TITLE
how do i make discord.py work with python 3.13? it cannot import audioop

### DISCUSSION THREAD

**Discussion by OLi51** [Rapptz/discord.py#10031] [Q&A] [answered]
i have been trying for a few hours now but I don't know how to make it not error
![image](https://github.com/user-attachments/assets/6dc19029-56df-4a4e-aa94-280f05985ff4)
I have tried installing audioop-lts and it's installed but it still cant import it

**Comment by MCausc78** [comment, marked as answer]
Try installing `master` branch (`pip install git+https://github.com/Rapptz/discord.py@master`)

**Reply by Yettyen** [reply]
Live saver! It worked like a charm!
Thank you so very much!💐✨
